lib/themes/dosomething/paraneue_dosomething/js/components/Button.js

```diff
@@ -0,0 +1,37 @@
+const React = require('react');
+
+/**
+ * Button Component
+ * <Button />
+ */
+const Button = (props) => {
+    const classes = ['button'].concat(props.classes);
+
+    if (props.loading === true) {
+      classes.push('is-loading');
+    }
+
+    if (props.success === true) {
+      classes.push('is-successful');
+    }
+
+    return (
+      <button className={classes.join(' ')} onClick={props.onClick} disabled={props.disabled}>{props.text}</button>
```

Comment:
I think you can probably just use `props.children` for the text, so the component can be used a bit more naturally: `<Button>Sign Up!</Button>`. And then you can validate that `props.children` must be a string in the PropTypes.